client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -147,4 +148,13 @@ class DomainRegistrationSuggestion extends React.Component {
 	}
 }
 
-export default localize( DomainRegistrationSuggestion );
+export default connect(
+	null,
+	( dispatch ) => {
+		return {
+			recordTracksEvent: ( event, properties ) => {
+				dispatch( recordTracksEvent( event, properties ) );
+			}
+		};
+	}
```

Comment:
fun fact!

this is quite verbose on the syntax and we can eliminate most of it…

```js
connect( null, { recordTracksEvent } )( localize( … ) );
```